incubator/druid/values.yaml

```diff
@@ -96,7 +96,7 @@ coordinator:
   config:
     DRUID_XMX: 256m
     DRUID_XMS: 256m
-    druid_coordinator_balancer_strategy: cachingCost
+    druid_coordinator_balancer_strategy: cost
```

Comment:
It is better to remove the `druid_coordinator_balancer_strategy` setting

Reply:
ok, thanks #23072